<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>828</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>828</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>781</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>함초롬바탕</family>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>501</width>
       <height>631</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="usesScrollButtons">
         <bool>true</bool>
        </property>
        <property name="documentMode">
         <bool>true</bool>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>인기메뉴</string>
         </attribute>
         <widget class="QScrollArea" name="scrollArea_7">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>491</width>
            <height>601</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_1">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>489</width>
             <height>599</height>
            </rect>
           </property>
           <widget class="QWidget" name="gridLayoutWidget_14">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>491</width>
              <height>601</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_8"/>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>시즌메뉴</string>
         </attribute>
         <widget class="QScrollArea" name="scrollArea_8">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>491</width>
            <height>601</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>489</width>
             <height>599</height>
            </rect>
           </property>
           <widget class="QWidget" name="gridLayoutWidget_15">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>491</width>
              <height>601</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_9"/>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>커피</string>
         </attribute>
         <widget class="QScrollArea" name="scrollArea_9">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>491</width>
            <height>601</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>489</width>
             <height>599</height>
            </rect>
           </property>
           <widget class="QWidget" name="gridLayoutWidget_16">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>491</width>
              <height>601</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_10"/>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>논커피</string>
         </attribute>
         <widget class="QScrollArea" name="scrollArea_10">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>491</width>
            <height>601</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>489</width>
             <height>599</height>
            </rect>
           </property>
           <widget class="QWidget" name="gridLayoutWidget_17">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>491</width>
              <height>601</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_11"/>
           </widget>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>스무디</string>
         </attribute>
         <widget class="QScrollArea" name="scrollArea_11">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>491</width>
            <height>601</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_5">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>489</width>
             <height>599</height>
            </rect>
           </property>
           <widget class="QWidget" name="gridLayoutWidget_18">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>491</width>
              <height>601</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_12"/>
           </widget>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>650</y>
       <width>621</width>
       <height>131</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QPushButton" name="allDel_btn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>전체삭제</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>431</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTableWidget" name="cart_tbl">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>320</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="edit_btn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>수정</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pay_btn">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>101</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>결제</string>
      </property>
     </widget>
     <widget class="QPushButton" name="del_btn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>삭제</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="home_btn">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>90</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>홈으로</string>
     </property>
    </widget>
    <zorder>groupBox_2</zorder>
    <zorder>verticalLayoutWidget</zorder>
    <zorder>home_btn</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
